package.json

```diff
@@ -11,15 +11,15 @@
     "url": "https://github.com/mobify/nightwatch-commands/issues"
   },
   "devDependencies": {
-    "nightwatch": "0.8.6",
+    "nightwatch": "0.9.1",
     "nodeunit": "~0.8.4",
     "grunt": "0.4.5",
     "grunt-eslint": "^17.0.0",
     "mobify-code-style": "^2.3.6"
   },
   "scripts": {
     "install": "node selenium/installation/install.js",
-    "test": "grunt lint"
+    "test": "grunt lint; grunt test"
```

Comment:
Good to know @mikenikles . However I do believe my point re. `grunt-cli` needing to be a `package.json` dep still applies.